src/lib/Hydra/Config.pm

```diff
@@ -0,0 +1,11 @@
+package Hydra::Config;
+
+use strict;
+use Exporter;
+
+our @ISA = qw(Exporter);
+our @EXPORT = qw(%configGeneralOpts);
+
+my %configGeneralOpts = (-UseApacheInclude => 1, -IncludeAgain => 1);
```

Comment:
my perl friend suggests:

```suggestion

our %generalOptions = (-UseApacheInclude => 1, -IncludeAgain => 1);
```

then in other places you can refer to %Hydra::Config::generalOptions`

also referred to https://metacpan.org/pod/Exporter#What-Not-to-Export

Reply:
Do I still need to `use Hydra::Config;`?

Reply:
I think yes, but worth an experiment.

Reply:
yeah no need to `use`. also it looks like adding `-IncludeRelative` made absolute paths work so I updated the doc.
also added a rudimentary-ish test that verifies nested includes work.
I think we're good.